app/views/users/_user.html.erb

```diff
@@ -0,0 +1,13 @@
+<article class="media is-align-items-center">
+  <div class="media-left">
+    <figure class="image is-128x128">
+    <%= image_tag user.photo %>
+    </figure>
+  </div>
+  <div class="media-content notification">
+    <div class="content">
+      <h4 class="title is-4"><%= user.name %></h4>
+      <p class="has-text-right is-size-7"><%= t('Posts_Counter', count: user.Posts_Counter)%></p>
+    </div>
+  </div>
+</article>
```

Comment:
- [ ] Style does not matter in this milestone. But at least change the layout like in the wireframe.
- [ ] Also ı cant see the post counter number on this page. Please fix it.
![image](https://user-images.githubusercontent.com/38635158/161262823-2b069022-8512-4168-8d04-f25fab186a6b.png)

Reply:
done